packages/sanity/src/core/releases/tool/components/Table/Table.tsx

```diff
@@ -195,7 +195,7 @@ const TableInner = <TableData, AdditionalRowTableData>({
               height: `${datum.virtualRow.size}px`,
               transform: `translateY(${datum.virtualRow.start - datum.index * datum.virtualRow.size}px)`,
               paddingInline: `max(
-                calc((100vw - var(--maxInlineSize)) / 2),
+                calc((100% - var(--maxInlineSize)) / 2),
```

Comment:
This is not using anymore always the `100vw` given the history inspector could take some of the viewport width.